WordPress/UITestsFoundation/Screens/ReaderScreen.swift

```diff
@@ -144,46 +154,55 @@ public class ReaderScreen: ScreenObject {
         (try? ReaderScreen().isLoaded) ?? false
     }
 
-    public func openDiscoverTab() -> Self {
-        discoverButton.tap()
-
-        return self
-    }
-
     public func selectTopic() -> Self {
         topicCellButton.firstMatch.tap()
 
         return self
     }
 
-    public func openSavedTab() -> Self {
-        savedButton.tap()
-
-        return self
-    }
-
     public func verifyTopicLoaded(file: StaticString = #file, line: UInt = #line) -> Self {
         XCTAssertTrue(readerButton.waitForExistence(timeout: 3), file: file, line: line)
         XCTAssertTrue(followButton.waitForExistence(timeout: 3), file: file, line: line)
 
         return self
     }
 
-    public func openFollowingTab() -> Self {
-        followingButton.tap()
+    public func followTopic() -> Self {
+        _ = followButton.waitForExistence(timeout: 3)
+        followButton.tap()
```

Comment:
```suggestion
        waitForExistenceAndTap(followButton, timeout: 3)
```

Hey you considered using [`waitForExistenceAndTap(_ element: XCUIElement, timeout: TimeInterval = 5)`](https://github.com/wordpress-mobile/WordPress-iOS/blob/76981b0569881be02ec1b3b9fb7f0288338d4a95/WordPress/UITestsFoundation/Globals.swift#L39)? It looks like a good case for that. =]

Reply:
Thank you! Fixed in https://github.com/wordpress-mobile/WordPress-iOS/pull/22469/commits/ead4d8f1f2a7c97d5153a0339b4a6f7455aac71a.